src/models/log.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Exercise } from "./exercise.model";
+
+export interface Log {
+  createdAt: Date;
+  exercises: Exercise[];
```

Comment:
If you agree with my above suggestion
```suggestion
  exercises: ExerciseLog[];
```